packages/core/ios/Sources/DdSdkNativeInitialization.swift

```diff
@@ -150,6 +150,11 @@ public class DdSdkNativeInitialization: NSObject {
             }
         }
         
+        var appHangThreshold: Double? = nil
+        if let customAppHangThreshold = configuration.appHangThreshold {
+            appHangThreshold = customAppHangThreshold
+        }
```

Comment:
Any reason you can't just do this?

```suggestion
        let appHangThreshold = configuration.appHangThreshold
```

In fact, why can't you read appHangThreshold directly below?

Reply:
I don't know how I missed that, you are right 😅